assignment4.js

```diff
@@ -0,0 +1,14 @@
+for(let i = 1;i<100;i++){
```

Comment:
this will not work accordingly

Reply:
Yes, there were some logic errors, I have made some changes and pushed the codde.